dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 nose==1.3.3
+NoseGAE==0.5.7
```

Comment:
So, can we move this into the tox.ini section for gae and make `test-gae` do

```
    tox -e gae
```

Or maybe,

```
    TOXENV=gae tox
```